change/@fluentui-react-charting-e4dc8ea7-f277-4bf1-a526-a85995f9f5e0.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30346&commitId=23e53cb0c5612eee8e2631c96d5cc90c8e1d3c2d&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-HorizontalBarChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HorizontalBarChart.With_Axis - RTL.default.chromium.png|34479| Changed|
|react-charting-HorizontalBarChart.With_Axis.default.chromium.png|34492| Changed|
|react-charting-HorizontalBarChart.With_Axis - Dark Mode.default.chromium.png|15526| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>